app-interface/manifests/rhose-fleet-manager.yml

```diff
@@ -201,6 +206,13 @@ objects:
                     secretKeyRef:
                       name: ${FLEET_MANAGER_SECRET_NAME}
                       key: SHARD_ID
+              volumeMounts:
+              - name: rds
+                mountPath: /secrets/rds
+          volumes:
+          - name: rds
+            secret:
+              secretName: ${RDS_SECRET_NAME}
               ports:
```

Comment:
Is it properly aligned? I would expect the content below to be part of `containers` block, not `volumes`

Reply:
@sutaakar looks ok but I'll check it out.


```
 oc apply -f app-interface/manifests/rhose-fleet-manager.yml --dry-run
W0516 12:59:08.669021   83255 helpers.go:555] --dry-run is deprecated and can be replaced with --dry-run=client.
W0516 12:59:09.551338   83255 shim_kubectl.go:55] Using non-groupfied API resources is deprecated and will be removed in a future release, update apiVersion to "template.openshift.io/v1" for your resource
template.template.openshift.io/rhose-fleet-manager created (dry run)
```

Reply:
@sutaakar https://medium.com/avmconsulting-blog/secrets-management-in-kubernetes-378cbf8171d0  I think it's ok, right? In this case I'm wanting to mount the secret as a volume.

Reply:
I am referring to this block:
```
          volumes:
          - name: rds
            secret:
              secretName: ${RDS_SECRET_NAME}
              ports:
                - containerPort: 8080
              livenessProbe:
                httpGet:
                  path: /q/health/live
                  port: 8080
                initialDelaySeconds: 10
                periodSeconds: 10
              readinessProbe:
                httpGet:
                  path: /q/health/ready
                  port: 8080
                initialDelaySeconds: 10
                periodSeconds: 10
```
When you apply the template then ports and probes are gone.

Reply:
@sutaakar got it! tks